contracts/PositionManager.sol

```diff
@@ -97,7 +97,7 @@ contract PositionManager is FeeCollector, IPositionManager {
     /// @param maxFeePercentage The max fee percentage to check.
     /// @param condition If true, the check will be performed.
     modifier validMaxFeePercentageWhen(uint256 maxFeePercentage, bool condition) {
-        if (condition && (maxFeePercentage < borrowingSpread || maxFeePercentage > MathUtils._100_PERCENT)) {
```

Comment:
Why delete `maxFeePercentage < borrowingSpread `?

Reply:
It wastes gas for regular usage. For failed vase it will fail anyway in _checkValidFee.